actions/destinations/update.go

```diff
@@ -15,10 +15,7 @@ import (
 // @Description	Update destination
 // @Tags		Destinations
 // @Produce		json
-// @Param		id path string false "Destination ID"
-// @Param		address path string false "Destination Address"
-// @Param		locking_script path string false "Destination Locking Script"
-// @Param		metadata body string true "Destination Metadata"
+// @Param		UpdateDestination body UpdateDestination false "UpdateDestination model containing an id, address, locking_script and metadata"
```

Comment:
Maybe we should remove the description on all update request bodies.
Because I don't have any good description for that.
```suggestion
// @Param		UpdateDestination body UpdateDestination false ""
```

Reply:
Done